distributions/2025-04-01/TACoRewardsDetails/DKGHeartbeatRituals.json

```diff
@@ -234,5 +234,234 @@
     "103": [
         "0x5270BA998643bE976c106298C08C46947c5Bf774",
         "0xA5A51DC547c7d0689a789Db0144C38CF71eD87F1"
+    ],
```

Comment:
Any reason 104 is not included?

Reply:
> Any reason 104 is not included?

Ritual 104 was a test ritual David and I ran to check the status of those three specific nodes, but it was not part of any DKG heartbeat round.